src/main/kotlin/uk/gov/dluhc/printapi/service/aed/AnonymousElectorDocumentService.kt

```diff
@@ -28,25 +28,25 @@ class AnonymousElectorDocumentService(
     fun generateAnonymousElectorDocument(eroId: String, dto: GenerateAnonymousElectorDocumentDto): PdfFile {
         verifyGssCodeIsValidForEro(eroId, dto.gssCode)
         val filename = pdfTemplateDetailsFactory.getTemplateFilename(dto.gssCode)
-        with(anonymousElectorDocumentMapper.toAnonymousElectorDocument(dto, filename)) {
+        with(generateAnonymousElectorDocumentMapper.toAnonymousElectorDocument(dto, filename)) {
             val templateDetails = pdfTemplateDetailsFactory.getTemplateDetails(this)
             val contents = pdfFactory.createPdfContents(templateDetails)
             anonymousElectorDocumentRepository.save(this)
             return PdfFile("anonymous-elector-document-$certificateNumber.pdf", contents)
         }
     }
 
-    fun getAnonymousElectorDocumentSummaries(
+    fun getAnonymousElectorDocuments(
```

Comment:
Sorry I think I should have done it earlier, but are you ok to add this bit ? 

@Transactional(readOnly = true)

Reply:
Good spot, done.